CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.1.3 (2020-02-12)
+
+- Fixed an Angular compiler template checking error on the tokens component HTML template. [#55](https://github.com/blackbaud/skyux-indicators/pull/55)
```

Comment:
We should probably make this "component's HTML template" similar to the other changelog entry.

Reply:
Done!